src/components/DeleteAccount.jsx

```diff
@@ -32,10 +32,10 @@ const DeleteAccount = ({ email }) => {
     setStatus(EMAIL_CONFIRMATION)
   }
 
-  const { canAuthWithOIDC } = useCanAuthWith()
+  const withoutPassword = useWithoutPassword()
 
   const onClick = () => {
-    if (canAuthWithOIDC) {
+    if (withoutPassword) {
```

Comment:
What is the expected behavior here?

If an OIDC user defines a password, do we expect the user to enter their password on deleting account? Or is the OIDC login enough?